### PR TITLE
Added checks against the use of the special uploader tokens

### PR DESCRIPTION
**Pull request by shamsasari**
"app", "rpc", "p2p" and "unknown" have security implications (see isUploaderTrusted method) and thus they are not allowed to be used in the uploader field when importing attachments via the public API.

### CODE REVIEW

**Comment by shamsasari** [comment]
I also removed "test" as a trusted uploader. Seems like a risky thing to have.

File: testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/InternalMockNetwork.kt [hunk 1]
```diff
@@ -547,6 +547,8 @@ open class InternalMockNetwork(defaultParameters: MockNetworkParameters = MockNe
         messagingNetwork.stop()
     }
 
+    override fun close() = startNodes()
```
Comment (tudor-malene) [inline testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/InternalMockNetwork.kt]: Shouldn't this be stopNodes?

Comment (shamsasari) [reply testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/InternalMockNetwork.kt]: LOL!

**Comment by shamsasari** [comment]
Should be good now.

**Pull request merged**